### ISSUE TITLE
Bug: Calling a specific tiprack erases calibration data for that tiprack

### ISSUE THREAD

**Issue by arachnologist** [Opentrons/opentrons#2077]
bug

 Not sure if anyone else is experiencing this problem, but basically defining what tiprack you want the multihead pipette to pick up tips from (not assigning tipracks in the pipette's definition) seems to erase said tiprack's calibration data. When running the protocol, the pipette is way off usually picking up only 2 tips while ramming into whatever labware is in the neighboring slot. This is mostly an issue when trying to design protocols that minimize how many times the pipette unnecessarily moves over DNA samples and therefore prevent cross-contamination.

App version: 3.2.0; firmware: edge-6168d32; software: 3.2.0
Basically this is fine:
    pipette_multi_50 = instruments.P50_Multi(
    tip_racks=[tips_1_rack_p100, tips_2_rack_p100, tips_3_rack_p100, tips_4_rack_p100],
    mount='left',
    aspirate_flow_rate=200,
    dispense_flow_rate=600
     )
But this is where you get the issue:
    pipette_multi_50.pick_up_tip(tips_2_rack_p100)
resulting in the pipette to be way off when trying to move to 'tips_2_rack_p100'

**Comment by arachnologist** [comment]
Really hoping this will get looking into soon. Our OT-2 is useless for assays requiring many tips (basically all of them) until we get this fixed; I've already alerted the Opentrons support team twice and never got a reply.

**Comment by Laura-Danielle** [comment]
Hi @arachnologist thank you for your submission!

This is not a bug, it is expected behavior. You are not giving the pipette an exact location to pick up tips from, instead you are passing in the whole container which causes the pipette to move to an assumed `top` location of that given container.

If you do the following:
`pipette_multi_50.pick_up_tip(tips_2_rack_p100.cols('1'))`

This will tell the pipette to pick up tips from the first column of the tiprack (for example).
